samples/05_content_publishers/overwriting_feature_layers.ipynb

```diff
@@ -330,7 +330,8 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    csv_item = root_folder.add(item_properties=item_prop, file=my_csv)
</pre><ol><li>need&nbsp;<code>item_type</code>&nbsp;as required key pair in item_properties</li><li>need&nbsp;<code>.result()</code>&nbsp;for&nbsp;<code>Folder.add()</code>&nbsp;job result</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>